artifactory_cleanup/rules/docker.py

```diff
@@ -110,11 +110,11 @@ def __init__(self, *, days: int):
 
     def aql_add_filter(self, filters):
         last_day = self.today - self.days
-        filter_ = {
+        filter_ = [
             {"stat.downloads": {"$eq": None}},
-            {"created": {"$lte": last_day.isoformat()}},
-        }
-        filters.append(filter_)
+            {"created": {"$lte": "x"}},
```

Comment:
I'm not sure that I get it - what does `"x"` mean here? Shouldn't it be `last_day.isoformat()`?

Reply:
oh yes, this is some classic example of "you are trying to debug something on your PR branch but forget to delete it before committing" it... :(
I'm really sorry! Should be fixed now, please squash my work while merging it...

Reply:
@StevenKGER I think it's the project fault - there's no tests for the method :(